pkg/term/term_windows.go

```diff
@@ -83,11 +83,13 @@ func useNativeConsole() bool {
 		return false
 	}
 
-	// TODO Windows. The native emulator still has issues which
-	// mean it shouldn't be enabled for everyone. Change this next line to true
-	// to change the default to "enable if available". In the meantime, users
-	// can still try it out by using USE_NATIVE_CONSOLE env variable.
-	return false
+	// Must have a post-TP5 RS1 build of Windows Server 2016/Windows 10 for
```

Comment:
Are there plans to provide post-TP5 through Windows Update or a "Inner ring" like Windows 10?